cmd/init.go

```diff
@@ -68,14 +68,15 @@ var InitCmd = &cobra.Command{
 }
 
 func init() {
-	InitCmd.Flags().BoolVarP(&kubernetesMode, "kubernetes", "k", false, "Deploy Dapr to a Kubernetes cluster")
-	InitCmd.Flags().BoolVarP(&slimMode, "slim", "s", false, "Initialize dapr in self-hosted mode without placement, redis and zipkin containers.")
-	InitCmd.Flags().StringVarP(&runtimeVersion, "runtime-version", "", "latest", "The version of the Dapr runtime to install. for example: v0.1.0")
+	InitCmd.Flags().BoolVarP(&kubernetesMode, "kubernetes", "k", false, "Deploys Dapr to a Kubernetes cluster")
+	InitCmd.Flags().BoolVarP(&slimMode, "slim", "s", false, "Excludes placement service, Redis and Zipkin containers from self-hosted installation")
```

Comment:
Should be capital P "Excludes Placement service..."

Reply:
We refer to "placement service" in lowercase everywhere in the cli code, including READMEs too.